src/stactools/cop_dem/commands.py

```diff
@@ -19,8 +19,10 @@ def cop_dem():
     @click.option("--validate/--no-validate",
                   default=True,
                   help="Validate the item before saving")
-    def create_item_command(source, destination, validate):
-        item = stac.create_item(source)
+    @click.option("--host", default=None, help="Set PROVIDER HOST")
```

Comment:
Do you want to limit accepted values to the keys in `co.COM_DEM_HOST`? If so, consider this (with required import as well):

```suggestion
    @click.option("--host",
                  type=click.Choice(co.COP_DEM_HOST, case_sensitive=False),
                  default=None,
                  help="Set PROVIDER HOST")
```

However, offhand, I don't know if `click` allows such choice options to be optional (i.e., with `default=None`). In other words, I don't know if specifying choices forces the option to be required, such that any value supplied as the default must be one of the choices (and thus cannot be None, unless perhaps None is one of the choices).

Reply:
We can test that idea.

Reply:
Also going to note that `click.Choice` wants a list so we need to get the keys from the Dict of possible hosts, not the dict.